source/supported-operations.txt

```diff
@@ -331,6 +344,9 @@ Date and Time Functions
      - Return the last day of the month for the argument
    * - ``MAKEDATE()``
      - Create a date from the year and day of year
+   * - ``MICROSECOND()``
+     - Return the microseconds from the time or datetime expression 
+       ``<expr>`` as a number in the range from 0 to 999999.
```

Comment:
Where does ``<expr>`` come from? Can we just say

> Return the microseconds from the input time or datetime expression as a number...

Reply:
Updated.